uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-balance-tab.component.js

```diff
@@ -98,15 +98,17 @@ define([
     /**
      * Loads a specific page of the report.
      *
-     * @param {int} [pageNumber=1] - The number of the page to retrieve. Defaults to 1.
      * @return {Promise}
      */
     function loadReportCurrentPage () {
       vm.loading.report = true;
 
       return LeaveBalanceReport.all(
-        filters,
-        vm.pagination
+        {
+          period_id: filters.absence_period,
+          type_id: filters.absence_type,
+          managed_by: filters.managed_by
+        }, vm.pagination
```

Comment:
why not change the filter properties so you can pass *filters* instead of creating a new object? (just like it was before)

Reply:
Please have a look at my reply here https://github.com/civicrm/civihr/pull/2133#discussion_r138408228